From ff4c7bc3bf7ba4b748fdeb9d09887271c2b28505 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Fri, 17 Sep 2021 10:14:42 +0200
Subject: [PATCH 3/3] Drop obsolete QtOpengl dependency

Widgets were ported away from QtOpenGL in 5.15.0 but bogus dependency
remained (commit 30034a140ca8aefa1986c9964ae1f30dcfef886e).

See also: https://bugreports.qt.io/browse/QTBUG-81902

qmake backport of upstream dev branch commit a7621a6db7bdbe514be825cbc2952d50e328bab4

Thanks-to: Davide Pesavento <pesa@gentoo.org>
Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 src/multimediawidgets/multimediawidgets.pro | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/multimediawidgets/multimediawidgets.pro b/src/multimediawidgets/multimediawidgets.pro
index 1919e8107..4c30d8fbf 100644
--- a/src/multimediawidgets/multimediawidgets.pro
+++ b/src/multimediawidgets/multimediawidgets.pro
@@ -2,8 +2,6 @@
 TARGET = QtMultimediaWidgets
 QT = core gui multimedia widgets-private
 QT_PRIVATE += multimedia-private
-qtHaveModule(opengl): \
-   QT_PRIVATE += opengl
 
 PRIVATE_HEADERS += \
     qvideowidget_p.h \
-- 
2.40.0.rc2

